dataduct/utils/cli.py

```diff
@@ -1,8 +1,24 @@
 """Helper function for CLI scripts
 """
+import argparse
+import contextlib
+import os
+
 from argparse import ArgumentParser
 from argparse import RawTextHelpFormatter
-import argparse
+
+
+@contextlib.contextmanager
+def chdir(dirname=None):
```

Comment:
because you only use this for command line inputs right ? otherwise I don't see why you should be changing directories.